### PR TITLE
Store user info in cookie store

### PR DESCRIPTION
**Pull request by Jasopaum**
## Fixes

Fixes #[issue number] by @[issue author]

### CODE REVIEW

File: provider/consent.go [hunk 1]
```diff
@@ -111,20 +103,27 @@ func (ctx *Provider) grantScopes(grantedScopes []string, consentChallenge string
...
 	putUrl := fmt.Sprintf("%s/accept?%s", ctx.HConf.ConsentRequestRoute, params.Encode())
 
-	// TODO use session to add info about the current user
-	session := SessionInfo{
+	// Use cookies to add info about the current user
+	session, err := ctx.Store.Get(r, sessionName)
+	if err != nil {
+		http.Error(w, errors.Wrap(err, "Error while reading cookies store").Error(), http.StatusInternalServerError)
+	}
+	userName, _ := session.Values["userName"]
```
Comment (simonvadee) [inline provider/consent.go:111]: why is the secon dvalue ignored?

**Review by simonvadee — APPROVED** [review]

File: provider/consent.go (hunk 1 above)
Comment (Jasopaum) [reply provider/consent.go:111]: The second value is a bool telling if the value was found or not, I wasn't sure what to do with it. What do you think?

Comment (simonvadee) [reply provider/consent.go:111]: either you ignore it like this:
```
userName := session.Values["userName"]
```
or you use the boolean to return an error if username/email was not found

Comment (simonvadee) [reply provider/consent.go:111]: depends if you think there is ANY chance that these variables could be missing from the session

**Pull request merged**